<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>468</width>
    <height>196</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Chirascan</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>ap.ico</normaloff>ap.ico</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="wlLabel">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>10</y>
      <width>47</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>nm</string>
    </property>
   </widget>
   <widget class="QPushButton" name="goWlButt">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>70</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Go</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="shutButt">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>70</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Shutter</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="bwLabel">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>40</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Bandwidth:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="bwEdit">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>70</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="setBwButt">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>70</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Set</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="cuvTempLabel">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>123</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="monoTempLabel">
    <property name="geometry">
     <rect>
      <x>116</x>
      <y>123</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="mtLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Monochr. temp.:</string>
    </property>
   </widget>
   <widget class="QLabel" name="ctLabel">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>120</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Cuvette temp.:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="wlEdit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>20</pointsize>
     </font>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>468</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
